x/application/client/cli/helpers_test.go

```diff
@@ -0,0 +1,28 @@
+// Package cli_test provides unit tests for the CLI functionality.
+package cli_test
+
+import (
+	"pocket/cmd/pocketd/cmd"
+	"pocket/testutil/network"
+	"pocket/x/application/types"
+	"strconv"
+	"testing"
```

Comment:
nit: (can/should we add a linter for this?)
```suggestion
	"strconv"
	"testing"
	
	"pocket/cmd/pocketd/cmd"
	"pocket/testutil/network"
	"pocket/x/application/types"
```